data/json/recipes/other/tool.json

```diff
@@ -128,5 +128,29 @@
       [ [ "pot_copper", 1 ] ],
       [ [ "hose", 2 ] ]
     ]
+  }, 
+  {
+    "result": "food_processor",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "skills_required": [ "fabrication", 4 ],
```

Comment:
I don't know if the difficulty is a modifier for crafting success, so you made left me thinking. Decided to check the recipe for the coffeemaker recipe and copied it's crafting, as I think it is more or less the same difficulty.